src/server.js

```diff
@@ -58,6 +58,11 @@ module.exports = function() {
 		log.warn("Server is public and set to use LDAP. Set to private mode if trying to use LDAP authentication.");
 	}
 
+	if (config.prefetchMaxImageSize) { // TODO: Remove in v3.0.0
+		log.warn(`${colors.bold("prefetchMaxImageSize")} is ${colors.bold.red("deprecated")} and will be removed in The Lounge v3.0.0. Use the ${colors.bold("prefetchMaxMediaSize")} option instead.`);
+		config.prefetchMaxMediaSize = config.prefetchMaxImageSize;
```

Comment:
Should we maybe do

`config.prefetchMaxMediaSize = config.prefetchMaxMediaSize || config.prefetchMaxImageSize`

Just incase any users have the media size set as well as the image size? I'm not sure what the best thing to do it, here.

Reply:
That seems good. Will do

Reply:
See above, there's currently no reason to change the server option (and it's misleading to change since it still will only affect images).